frontend/src/pages/EventsPage/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { withStyles } from '@material-ui/core/styles';
 import { compose } from 'recompose';
-import { Divider } from '@material-ui/core';
+import { HomepageBanner } from '../../components/HomepageBanner';
```

Comment:
```suggestion
import { HomepageBanner } from '@SharedComponents/HomepageBanner';
```

We have aliases setup [here](https://github.com/HKN-UCSD/hkn-server/blob/master/frontend/tsconfig.paths.json), plz try to use those because it makes moving files around later easier since we no long rely on relative path for imports.